app/models/feedback.rb

```diff
@@ -0,0 +1,2 @@
+class Feedback < ApplicationRecord
```

Comment:
@micapam its a good point but I don't think we need to consider it in this PR. I would like to extract the feedback tool into its own service at which point we could consider these sorts of issues.